wc_langPacks/wc_langPack_EN.js

```diff
@@ -45,9 +45,9 @@ wc_addLanguagePack({
     // 5+3, 5+4 => nearly
     if (
       (settings.fuzzyTime == 'both' || settings.fuzzyTime == 'after') &&
-      m % 5 <= 2
+      m % 5 <= 2 && m % 5 != 0
     ) {
-      ret += 'JUST WAS ';
+      ret += 'IS JUST AFTER ';
```

Comment:
Can you move the IS out to line 26?
Then you can remove the else.
Thanks for catching the 0 minutes bug.